src/main/java/lotto/Ticket.java

```diff
@@ -0,0 +1,26 @@
+package lotto;
+
+import java.util.ArrayList;
+import java.util.Collections;
+
+public class Ticket extends ArrayList<Integer> {
+
+    public Ticket() {
+        createNumbers();
+    }
+
+    public Ticket(ArrayList<Integer> testInput) {
+        this.addAll(testInput);
+    }
+
+    private void createNumbers() {
+
+        for (int i = 1; i <= 45; i++) {
+            this.add(i);
+        }
+
+        Collections.shuffle(this);
+        this.subList(6, 45).clear();
+        Collections.sort(this);
+    }
```

Comment:
```suggestion
    private List<Integer> ticketNumbers;
    
    public Ticket() {
        this.ticketNumbers = createNumbers();
    }

    public Ticket(ArrayList<Integer> testInput) {
        this.ticketNumbers = testInput;
    }

    private List<Integer> createNumbers() {
        List<Integer> numbers = new ArrayList<>();
        
        for (int i = 1; i <= 45; i++) {
            numbers.add(i);
        }

        Collections.shuffle(numbers);
        numbers.subList(6, 45).clear();
        return Collections.sort(numbers);
    }
```
이런 형태이면 상속을 제거해도 되겠죠? 상속을 사용했을 때의 단점은 매우 강합니다. 
예를 들어, List가 아닌 Set 자료구조를 사용하라는 요구사항이 발생했을 경우에는 변경해야 할 사항들과 고려해야 할 사항들이 많아지거든요.
상속에 대해 공부해보면 좋을 것 같아요.